dali/CMakeLists.txt

```diff
@@ -98,6 +99,9 @@ if (BUILD_DALI_PIPELINE AND BUILD_TEST)
   if (BUILD_CUFILE)
     target_link_libraries(dali_test PRIVATE dynlink_cufile)
   endif()
+  if (BUILD_LIBTAR)
+    target_link_libraries(dali_test PRIVATE libtar)
+  endif()
```

Comment:
I believe we shouldn't need `libtar` linked to `dali_test` target. I guess you linked it, because the test wasn't running properly. `libtar` is linked to `dali_operators` target below. `dali_operators` is linked to `dali_operator_test`, where the `LibTar` test might exist.

Reply:
I linked it because I didn't want to link it publically to dali_operators